multimedia/gstreamer1-plugins/Makefile

```diff
@@ -17,6 +17,7 @@ MAINTAINER=	multimedia@FreeBSD.org
 COMMENT?=	GStreamer written collection of plugins handling several media types
 
 BUILD_DEPENDS+=	gstreamer1>=${GST1_VERSION}${GST1_MINIMAL_VERSION}:multimedia/gstreamer1 \
+		${LOCALBASE}/include/linux/input.h:devel/evdev-proto \
```

Comment:
For all plugins? What does depend on this?

Reply:
You are right only gstream1-plugns-gl need this